slices/libatomic1.yaml

```diff
@@ -0,0 +1,10 @@
+package: libatomic1
+
```

Comment:
We need a `copyright` slice pointing to the license in the package as well as a `essential` field in the root containing the copyright slice.

I have an example here you can follow: https://github.com/canonical/chisel-releases/blob/ubuntu-22.04/slices/libcurl4.yaml#L4 

We also need these changes in slices/libtbb2.yaml and slices/libtbbmalloc2.yaml

Reply:
Hi @clay-lake,

I've added the copyright items to slices/libtbb2.yaml and slices/libtbbmalloc2.yaml according to your example, hope it's what you expected.

cheers,
alex

Reply:
I think Clay meant something like below:

```yaml
package: libatomic1

essential:
  - libatomic1_copyright

slices:
  libs:
    essential:
      - libc6_libs
    contents:
      /usr/lib/*-linux-*/libatomic.so.1*:

  copyright:
    essential:
      - gcc-12-base_copyright
    contents:
      /usr/share/doc/libatomic1:
```

Reply:
Yup, you are right. Interesting that this went unnoticed by the CI.

If you need it urgently, please raise another PR. Should be very short. :)

Reply:
I'll submit a separate PR to fix `libncursesw6` for all branches then...